adagrams/game.py

```diff
@@ -1,13 +1,161 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    #make a new list named pool_list
+    pool_list = []
+    for letter, frequency in LETTER_POOL.items():
+        pool_list.extend(letter * frequency)
+    
+
+    #draw ten times from pool_list randomly and remove the tile from
+    #the pool_list
+    drawn_list = []
+    
+    draw_count = 10
+    for i in range(0, draw_count):
+        index = randint(0, len(pool_list) - 1)
+        element = pool_list.pop(index)
+        drawn_list.append(element)
+
+
+    return drawn_list    
 
 def uses_available_letters(word, letter_bank):
-    pass
+    # is_valid = False
+    # letters_copy = letter_bank[:]
+    
+    # for letter in word.upper():
+    #     if letter in letters_copy :
+    #         is_valid  = True
+    #         letters_copy.remove(letter)
+    #     else:
+    #         return False
+    # return is_valid 
```

Comment:
We want to remove commented out code from our PRs, especially in shared projects, so that we don't bring things into the codebase that could be confusing down the road. 

If we have been making commits as we work, we can always check out the code at those commits if we want to take a look at something we've since changed or deleted.